src/components/skeleton/skeleton_item.tsx

```diff
@@ -0,0 +1,51 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+import React, { FunctionComponent, HTMLAttributes } from 'react';
+import classNames from 'classnames';
+
+import { CommonProps } from '../common';
+import { useEuiTheme } from '../../services';
+
+import { euiSkeletonItemStyles } from './skeleton_item.styles';
+
+export const RADIUS = ['s', 'm', 'none'] as const;
+export type SkeletonItemRadius = typeof RADIUS[number];
+
+export type EuiSkeletonItemProps = HTMLAttributes<HTMLDivElement> &
+  CommonProps & {
+    width?: string;
+    height?: string;
+    radius?: SkeletonItemRadius;
```

Comment:
Because this affects the CSS `border-radius` property  the name should be more descriptive:

```suggestion
    borderRadius?: SkeletonItemRadius;
```

Reply:
Included also this in the `EuiSkeletonRectangle` commit https://github.com/elastic/eui/pull/6502/commits/8c0a3013c99bb9727fb1b48e4b5553792e478062